### PR TITLE
Fixed community tag

### PR DESCRIPTION
**Pull request by victormihalache**

### CODE REVIEW

File: locales.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
     "displayName": "Magyar"
   },
   "it_IT": {
-    "displayName": "Italiano"
+    "displayName": "Italiano",
+    "community: true,
```
Comment (19wintersp) [inline locales.json]: Trailing comma?

**Review by 19wintersp — CHANGES_REQUESTED** [review]
Please can you try to check that this definitely works. Standard JSON also does not permit trailing commas, and NodeJS (at least my installation) does not either.

**Comment by victormihalache** [comment]
> Please can you try to check that this definitely works. Standard JSON also does not permit trailing commas, and NodeJS (at least my installation) does not either.

Should now work, dunno how i missed that.

**Review by 19wintersp — APPROVED** [review]
As long as you've checked it works on a full build, this is fine.

**Review by Dreiparrent — APPROVED** [review]

**Pull request merged**